src/OpenTelemetry.Resources.AWS/README.md

```diff
@@ -20,18 +20,21 @@ dotnet add package OpenTelemetry.Resources.AWS
 ## Usage
 
 You can configure AWS resource detector to
-the `TracerProvider` with the following EC2 example below.
+the `ResourceBuilder` with the following EC2 example.
 
 ```csharp
 using OpenTelemetry;
 using OpenTelemetry.Resources;
 
-var tracerProvider = Sdk.CreateTracerProviderBuilder()
-                        // other configurations
-                        .SetResourceBuilder(ResourceBuilder
-                            .CreateEmpty()
-                            .AddAWSEC2Detector())
-                        .Build();
+using var tracerProvider = Sdk.CreateTracerProviderBuilder()
+    .ConfigureResource(resource => resource.AddAWSEC2Detector())
+    // other configurations
+    .Build();
```

Comment:
Could you please also add example for logs (all readme files)?

Reply:
Ref: #1509 and https://github.com/open-telemetry/opentelemetry-dotnet/pull/5188

Reply:
Done. Do you think this PR fixes #1509?

Reply:
Yes! Edited first comment to automatically close that issue.